File: music-api/src/main/java/com/music/sale/web/user/request/SetInterestedCategoriesRequest.java
package com.music.sale.web.user.request;

import jakarta.validation.constraints.NotEmpty;
import java.util.List;

public record SetInterestedCategoriesRequest(
        @NotEmpty(message = "관심 카테고리는 최소 1개 이상 선택해야 합니다")
        List<Long> categoryIds) {}

File: music-api/src/main/kotlin/com/music/sale/web/user/request/SetInterestedCategoriesRequest.kt
package com.music.sale.web.user.request

import jakarta.validation.constraints.NotEmpty

data class SetInterestedCategoriesRequest(
    @field:NotEmpty(message = "관심 카테고리는 최소 1개 이상 선택해야 합니다") val categoryIds: List<Long>,
)
